debugger.jpda/nbproject/project.properties

```diff
@@ -25,5 +25,6 @@ requires.nb.javac=true
 spec.version.base=1.108.0
 test-unit-sys-prop.test.dir.src=${basedir}/test/unit/src/
 test-unit-sys-prop.netbeans.user=${basedir}/work/nb_user_dir
+test.unit.cp.extra=../java.source.nbjavac/build/test-nb-javac/cluster/modules/org-netbeans-modules-java-source-nbjavac-test.jar
```

Comment:
A trick to allow java.source.nbjavac to be enabled in tests when running tests with NB module system. Already used by refactoring.java, but will need tweaking in the future to simplify running the tests on vanilla javac as well.

Reply:
True, this is nasty.